packages/devops/scripts/deployment/fetchEnvVarsFromLastpass.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -le
+
+DIR=$(dirname "$0")
+TUPAIA_DIR=$DIR/../../../..
+DEPLOYMENT_NAME=$1
+
+cd ${TUPAIA_DIR}
+
+# Initialise NVM (which sets the path for access to npm, yarn etc. as well)
+. $HOME/.nvm/nvm.sh
+
+# Inject environment variables from LastPass
+LASTPASS_EMAIL=$($DIR/fetchParameterStoreValue.sh LASTPASS_EMAIL)
+LASTPASS_PASSWORD=$($DIR/fetchParameterStoreValue.sh LASTPASS_PASSWORD)
+LASTPASS_EMAIL=$LASTPASS_EMAIL LASTPASS_PASSWORD=$LASTPASS_PASSWORD yarn download-env-vars $DEPLOYMENT_NAME
```

Comment:
Moved this logic out of `buildDeployablePackages.sh` so that the RDS password can be set without first building the packages